Packs/FeedAutofocus/Integrations/FeedAutofocus/README.md

```diff
@@ -1,6 +1,7 @@
 Use the AutoFocus Feeds integration to fetch indicators from AutoFocus.
 For more information click [here](https://docs.paloaltonetworks.com/autofocus/autofocus-admin/autofocus-feeds.html).
-The Daily Feed is deprecated. Use the AutoFocus Daily Feed instead. 
+This Feed supports the AutoFocus Custom Feed and the AutoFocus Samples Feed.
+To ingest the Daily Feed, Use the [AutoFocus Daily Feed](https://xsoar.pan.dev/docs/reference/integrations/auto-focus-daily-feed). 
```

Comment:
`To ingest the Daily Feed, use the [AutoFocus Daily Feed](https://xsoar.pan.dev/docs/reference/integrations/auto-focus-daily-feed)`